s3-source-connector/src/test/java/io/aiven/kafka/connect/s3/source/input/ByteArrayTransformerTest.java

```diff
@@ -54,7 +54,7 @@ void testGetRecordsSingleChunk() {
         final IOSupplier<InputStream> inputStreamIOSupplier = () -> inputStream;
 
```

Comment:
And a test here to skip some records.

Reply:
For bytes transformer, we actually are not skipping any records. I shall remove that skipRecords on bytes transformer. Earlier we used to split based on max message bytes, then we removed that and thought of handling it as a malformed record.